lib/rake/test.rake

```diff
@@ -185,6 +185,12 @@ namespace :test do
       end
     end
 
+    task :interpreter do
+      run_tests_if_changed('interpreter', ['apps/src/lib/tools/jsinterpreter/patchInterpreter.js']) do
+        TestRunUtils.run_interpreter_tests
+      end
```

Comment:
Well, I guess it takes more than 2 hours because circle is timing out. I don't think there is a particularly quick fix for that.